src/features/create-dao/steps/TreasuryForm.module.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../../../../zUI/src/styles/colors';
+
+.Footer {
+	margin-top: 5rem;
+}
+
+.InputExtraInfo {
+	text-align: right;
+	font-size: 0.75rem;
+	color: #{colors.$blue};
```

Comment:
Same here, does this not throw an error in the UI ? Oh I think I see why this would work, you should import these from the `zUI` package, rather than local linking?

Reply:
Yep, good shout. Have updated import!